src/components/graphQLConnect.ts

```diff
@@ -28,6 +28,27 @@ class GraphQLdb {
     return data.data
   }
 
+  public getReviewData = async (reviewId: number) => {
+    const data = await this.gqlQuery(
+      `
+      query getReviewData($id: Int!) {
+        review(id: $id) {
+          level
+          isLastLevel
+          status
+          trigger
```

Comment:
Don't really need the trigger here do we? (It should always be Processing when an Action is running)

Reply:
9c984f37500ebb53be2fce20bd71a82a58205398